graspy/simulations/sims_sample_corr.py

```diff
@@ -0,0 +1,91 @@
+import numpy as np
+from graspy.simulations import sample_edges
+import copy
+import warnings
+
+def sample_corr(P, Rho, directed=False, loops=False):
```

Comment:
I am wondering if we should call this `sample_edges_corr` for consistency with the other function, what do you think?

Reply:
I agree